package.json

```diff
@@ -21,6 +21,7 @@
     "redux-logger": "2.4.0",
     "redux-thunk": "1.0.3",
     "reselect": "2.0.3",
+    "seed-random": "^2.2.0",
```

Comment:
Me thinks drop the semver range operator (`^`) here for consistency with other deps above.